### PR TITLE
Fix/customization product text

### LINKED ISSUE #26374: FO- Bad display when entering a long text for product customization
#### Describe the bug

Bad display when entering a long text for product customization

#### Expected behavior

Newline and left alignment when entering a long text for product customization

#### Steps to Reproduce

1. Go to FO> choose customizable product 
2. In the Product customization part : enter a long text without space bar ( number consisting of 100 digits for exemple)
3. Click on "SAVE CUSTOMIZATION"
4. The length of the text entered will exceed the size of the table as shown in the attached picture
image

5. Product customization part : enter a long text with space bar separating words
6. Click on "SAVE CUSTOMIZATION"
7. The entered text is aligned to the right

image

#### Additional information

* PrestaShop version: 1.7.8.x, 1.7.7.8,develop
* PHP version: 7.3

### PR DESCRIPTION
**Pull request by marwachelly**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop 
| Description?      |  Bad display when entering a long text for product customization
| Type?             | bug fix
| Category?         | FO 
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #26374.
| How to test?      | Check the Newline and left alignment when entering a long text for product customization
| Possible impacts? | /

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25883)

### CODE REVIEW

**Review by matthieu-rolland — DISMISSED** [review]
lgtm 👍

**Comment by matks** [comment]
Hello @marwachelly it seems you have some unwanted commits on your branch
Capture d’écran 2021-10-22 à 16 01 30

File: themes/classic/_dev/css/components/products.scss [hunk 1]
```diff
@@ -755,6 +755,10 @@
 
   .customization-message {
     margin-top: 20px;
+    .customization-label {
```
Comment (NeOMakinG) [inline themes/classic/_dev/css/components/products.scss:758]:
```suggestion

    .customization-label {
```
Just a little readability improvement

**Comment by NeOMakinG** [comment]
Also please run `npm run scss-fix` inside `themes/classic/_dev` folder

**Comment by Progi1984** [comment]
@marwachelly Is it a duplicate for #26394 ?

**Comment by marwachelly** [comment]
Hello, 
Issue of commit is fixed and another PR is created. https://github.com/PrestaShop/PrestaShop/pull/26394

**Pull request closed without merging**